bundle-size/common.js

```diff
@@ -49,6 +49,66 @@ exports.getCheckFromDatabase = async (db, headSha) => {
  * @param {number} delta the bundle size delta in KB.
  * @return {string} formatted bundle size delta.
  */
-exports.formatBundleSizeDelta = delta => {
+const formatBundleSizeDelta = delta => {
   return 'Δ ' + (delta >= 0 ? '+' : '') + delta.toFixed(2) + 'KB';
 };
+
+/**
+ * @param {string} file
+ * @param {string} description
+ * @return {string}
+ */
+const formatFileItem = (file, description) => `* \`${file}\`: ${description}`;
+
+exports.formatFileItem = formatFileItem;
+
+/**
+ * @param {{file: string, bundleSizeDelta: number}} item
+ * @return {string}
+ */
+exports.formatBundleSizeItem = ({file, bundleSizeDelta}) => {
+  return formatFileItem(file, formatBundleSizeDelta(bundleSizeDelta));
+};
+
+/**
+ * @param {string} file
+ * @return {string}
+ */
+const noExtension = file => {
+  const parts = file.split('.');
+  parts.pop();
+  return parts.join('.');
+};
+
+/**
+ * @param {{file: string, bundleSizeDelta: number}[]} items
+ * @param {'asc'|'desc'} sizeOrder
```

Comment:
Nit: I doubt we'll ever really need both options, we can simplify the code by just assuming desc